lib/generateAllTypes.js

```diff
@@ -0,0 +1,23 @@
+const { UserInputError } = require('./errors');
+
+/**
+ * Helper for generating all available JSON schema types.
+ *
+ * @param {String} [exclude=''] Types to be excluded separated by comma
+ * @return {Array} Array of string.
+ */
+
+const generateAllTypes = ({ exclude = '' }) => {
+	if (typeof exclude !== 'string') {
+		throw new UserInputError(
+			'The type of "exclude" argument must be string.',
+		);
+	}
+	let types = ['string', 'number', 'object', 'array', 'boolean', 'null'];
+	const excludedTypesArr = exclude.split(',').map((type) => type.trim());
+	types = types.filter((type) => !excludedTypesArr.includes(type));
+
+	return types;
+};
```

Comment:
I'd provide a default empty object in the signature:
`const generateAllTypes = ({ exclude = '' } = {}) => {`
so that you can call it without passing in an object if there is noting to exclude:
`generateAllTypes()` rater than `generateAllTypes({})`